utils/api-scripts/src/initialize-lead.ts

```diff
@@ -8,7 +8,7 @@ const workingGroupModules = [
   'contentWorkingGroup',
```

Comment:
I think we can drop this entierly. It uses sudo. We forgot about it in mainnet-liberated branch